sixtyfps_runtime/interpreter/eval.rs

```diff
@@ -747,7 +747,10 @@ fn eval_assignment(lhs: &Expression, op: char, rhs: Value, local_context: &mut E
                         if op == '=' {
                             model.set_row_data(index, rhs);
                         } else {
-                            model.set_row_data(index, eval(model.row_data(index)));
+                            model.set_row_data(
+                                index,
+                                eval(model.row_data(index).unwrap_or_default()),
```

Comment:
I think this is similar to above, the fallback should be a non-void type:

```suggestion
                                eval(model.row_data(index).unwrap_or_else(|| default_value_for_type(lhs.type())),
```

Reply:
(Ignore this, in the light of the above discussion :-)